### DISCUSSION TITLE
Badge와 CommunityUser 모델에 대한 Composable 분리

### DISCUSSION THREAD

**Discussion by nutyworks** [implude/meat-in-android#55] [Ideas]
현재 디자인을 보면 아래와 같이 `뱃지 이름`이 동시에 나타는 경우가 많습니다. 중복을 줄이기 위해서 Badge와 CommunityUser composable을 따로 만드는 것은 어떨까요?
![image](https://user-images.githubusercontent.com/48619865/136508411-7d09f3e0-6919-40ec-941e-0aa79f39135f.png)
![image](https://user-images.githubusercontent.com/48619865/136508444-372c8344-c5e9-4693-bcb2-020410473d58.png)
![image](https://user-images.githubusercontent.com/48619865/136508469-adaa1530-d795-4bcc-8eb9-e28682e77b8c.png)

**Comment by jhk2004** [comment]
찬성합니다

**Comment by Gijuno** [comment]
저도 찬성합니다 더 효율적일듯 합니다
